populate.py

```diff
@@ -0,0 +1,61 @@
+import sys
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from config.db_setup import Base, Survey, Feedback, Answer
+
+if __name__ == '__main__':
+
+	if len(sys.argv) < 2:
+		print('Usage: python populate.py <postgres_url>')
+		print('Example postgres_url: postgresql+psycopg2://insi:somepasswd@localhost:5432/llb')
+		sys.exit()
+
+	else:
+		postgres_url = sys.argv[1]
+		try:
+			engine = create_engine(postgres_url)      # engine bound to type of database
+			Base.metadata.bind = engine               # base class (+ my classes) bound to engine
+			DBSession = sessionmaker(bind=engine)     # session class bound to engine
+			session = DBSession()                     # instance of session class
+
+			# Note:
+
+			# Session not persisted into db until:
+			# session.commit()
+
+			# If needed to revert all changes back to the last commit, call:
+			# session.rollback()
+
+
+			# Create entries
+			s = Survey(1, 'Magnificent survey')
```

Comment:
Could you explain this and the following lines? Are you testing the connection to the db by creating and updating records? If so, is it the best way? Or should we go with unit-testing option for this one?

Reply:
This is a diposable file I just used to check that the connection and ORM were working. I figured others might benefit from seeing the code so they can pick up the essentials of db manipulation via SQLAlchemy easily. (In case this project needs an ORM, as I think was the case)